File: backend/src/main/java/com/givers/repository/entity/Log.java
	package com.givers.repository.entity;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Document 
@Data 
@AllArgsConstructor
@NoArgsConstructor
public class Log {
	@Id
	String id;
	String username;
	String causeId;
	String eventType;
	String causeName;
	long created;
}

File: frontend/src/app/log.ts
export class Log {
    id: string;
    username: string;
    causeId: string;
    eventType: string;
    causeName: string;

    constructor(
        id: string,
        username: string,
        causeId: string,
        eventType: string,
        causeName: string
    ) {
        this.id = id;
        this.username = username;
        this.causeId = causeId;
        this.eventType = eventType;
        this.causeName = causeName;
    }
}
